eksconfig/add-on-managed-node-groups.go

```diff
@@ -143,6 +146,13 @@ type MNG struct {
 	VersionUpgrade *MNGVersionUpgrade `json:"version-upgrade,omitempty"`
 }
 
+// A MNGScaleConfig contains the minimum, maximum, and desired node counts for a nodegroup
+type MNGScaleConfig struct {
+	minSize     int `json:"min-size,omitempty"`
```

Comment:
Can we capitalize (e.g. `MinSize`...), otherwise, it won't be accessible outside of `eksconfig`